SAMtemplates/alarms/main.yaml

```diff
@@ -81,3 +85,24 @@ Resources:
         - !ImportValue lambda-resources:SlackAlertsSnsTopicArn
       OKActions:
         - !ImportValue lambda-resources:SlackAlertsSnsTopicArn
+
+  PSUConcurrencyAlarm:
+    Type: AWS::CloudWatch::Alarm
+    Properties: 
+      AlarmName: !Sub "${StackName}-Concurrency-Alarm"
```

Comment:
alarm names should be in the format "<stackname>_<resource>_<metric>". The underscores are important to allow the lambda to pull the stack name out properly including so it doesnt get confused with PR's that have - in the them. The rest it to give the alert an informative name when it comes through to slack, so think of it less as a traditional resource name and more a alarm display name (so you dont need alarm in it). 

So in this case it may want to be something like "${stackName}_Lambda_Concurrency".

Reply:
Excellent point, thanks. Fixed